src/radio-tile/radio-tile.scss

```
@@ -0,0 +1,143 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+@mixin iui-radio-tile {
+  cursor: pointer;
+
+  &:first-of-type > .iui-radio-tile-content {
+    border-radius: $iui-border-radius 0 0 $iui-border-radius;
+  }
+
+  &:last-of-type > .iui-radio-tile-content {
+    border-radius: 0 $iui-border-radius $iui-border-radius 0;
+  }
+
+  &:not(:first-of-type) > .iui-radio-tile-content {
+    margin-left: -1px;
+  }
+
+  &-content {
+    width: $iui-xl * 5;
+    height: 100%;
+    box-sizing: border-box;
+    padding: $iui-s;
+    position: relative;
+    z-index: 1;
+    border: 1px solid rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-4));
+    background-color: var(--iui-color-background-1);
+    @media (prefers-reduced-motion: no-preference) {
+      transition: border-color $iui-speed-fast ease-out;
+    }
+  }
+
+  &-container {
+    display: inline-flex;
+    flex-wrap: wrap;
+    user-select: none;
+    // Create stacking context
+    position: relative;
+    z-index: 0;
+  }
+
+  &-input {
+    width: 0;
+    height: 0;
+    appearance: none;
+    opacity: 0;
+    position: absolute;
+    @include focus-visible {
+      + * {
+        outline: 2px solid var(--iui-color-foreground-primary);
+        outline-offset: -4px;
```

```suggestion
        outline-offset: -4px;
        @media (forced-colors: active) {
          outline-color: Highlight;
        }
```
Adds support for `forced-colors`.